tools/scripts/internal-link-checker.js

```diff
@@ -0,0 +1,23 @@
+const { execSync } = require('child_process');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const internalLinks = process.env.INTERNAL_LINKS.split(',');
+
+for (const internallink of internalLinks) {
+  const result = execSync(
+    `grep --include="./dist/**.json" -rl "./dist" -e "${internallink}"`,
+  );
+  if (result.toString().length > 0) {
+    console.log(
+      `Failed because internal links were found in files:\n${result.toString()}`,
+    );
+    process.exit(1);
+  } else {
+    console.log('No internal links were found.');
+    process.exit(1);
+  }
```

Comment:
This will always break the CI pipeline. 
Please remove the else condition. it is unnecessary.